src/praisonai/praisonai/cli/commands/onboard.py

```
@@ -0,0 +1,22 @@
+
+@app.callback(invoke_without_command=True)
+def onboard_callback(ctx: typer.Context):
+    """Run the bot onboarding wizard."""
+    if ctx.invoked_subcommand:
+        return
+    from ..features.onboard import run_onboard
+    try:
+        run_onboard()
+    except KeyboardInterrupt:
+        get_output_controller().print_info("Cancelled.")
+        raise typer.Exit(130)
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)